static/js/pages/committee-single.js

```diff
@@ -383,6 +408,11 @@ $(document).ready(function() {
           columns: disbursementPurposeColumns,
           callbacks: aggregateCallbacks,
           order: [[1, 'desc']],
+          hideEmptyOpts: {
+            dataType: 'disbursements',
+            name: context.name,
```

Comment:
This is an old hound comment. I added it as a [global var](https://github.com/18F/openFEC-web-app/pull/1205/files#diff-93a270b5de8f2744399104a7ac896334R3) at the top of the file, but after the errors.